### PR TITLE
Execute steps in given order.

### LINKED ISSUE #664: Story call should execute steps in given order.

### PR DESCRIPTION
**Pull request by proofit404**
Resolves #664

### CODE REVIEW

**Comment by 0cry** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>PR should resolve related issue</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against bfd30b73d3a8e713a6b2bbfcdce6ee55dfc24a27
</p>

File: docs/story.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+As you may notice, we don't require any specific way to instantiate the class.
+It's up to you whether or not to use libraries like `attrs`, `dataclasses`,
+`pydantic` or use plain `__init__` method instead.
+
+=== "attrs"
+
+    ```pycon
+
+    >>> True
```
Comment (proofit404) [inline docs/story.md]: Provide examples of attrs story.

File: docs/index.md [hunk 2]
```diff
@@ -28,47 +28,47 @@ A line of text explaining snippet below…
 >>> from dataclasses import dataclass
```
Comment (proofit404) [inline docs/index.md:28]: Correct placeholder text for library description, pros features and code example.

File: docs/story.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+
+    ```
+
+### Steps executed in specified order
+
+=== "sync"
```
Comment (proofit404) [inline docs/story.md:141]: Add explanation text for this section.

**Pull request merged**